packages/botonic-plugin-flow-builder/tests/bot-action.test.ts

```diff
@@ -0,0 +1,83 @@
+import { INPUT } from '@botonic/core'
+import { describe, test } from '@jest/globals'
+
+import { BOT_ACTION_PAYLOAD_SEPARATOR } from '../src/constants'
+import { FlowBuilderAction, FlowText } from '../src/index'
+import { ProcessEnvNodeEnvs } from '../src/types'
+import { testFlow } from './helpers/flows'
+import {
+  createFlowBuilderPlugin,
+  createRequest,
+  getActionRequest,
+} from './helpers/utils'
+
+describe('Check the contents returned by the plugin', () => {
+  process.env.NODE_ENV = ProcessEnvNodeEnvs.PRODUCTION
+  const flowBuilderPlugin = createFlowBuilderPlugin(testFlow)
+
+  test('The user does the first interaction', async () => {
```

Comment:
Observation/Change: Try to add a more descriptive names for the test to explain what you're checking with this test.
For example, in this case, I would change it to something like this: "The starting content is displayed on the first interaction"